docs/faststore/docs/headless-cms/3-adding-content-types-and-sections.mdx

```diff
@@ -124,7 +124,14 @@ These components are referred to as **Sections**, and to create them,
 you can follow the instructions provided in the guide [Creating a new section](https://developers.vtex.com/docs/guides/faststore/building-sections-creating-a-new-section). 
 The guide offers detailed information on the process of creating Sections and integrating them into the Headless CMS for easy editing and management.
 
-#### The **`schema`** property
+#### The Children section
```

Comment:
suggestion: move this section to https://developers.vtex.com/docs/guides/faststore/headless-cms-overview